### ISSUE TITLE
ChestCommands disabled

### ISSUE THREAD

**Issue by Dark-LYNN** [filoghost/ChestCommands#156]
### Confirmation

- [X] I have read the [documentation](https://filoghost.me/docs/chest-commands).
- [X] I have read the [FAQ](https://filoghost.me/docs/chest-commands/faq).

### Description

i reloaded my server and now chestcommands is disabled 
How do i fix this?
error:
```console
[12:47:22] [Server thread/INFO]: [ChestCommands] Enabling ChestCommands v4.0.3
[12:47:22] [Server thread/INFO]: [ChestCommands] Fatal unexpected error while enabling plugin:
java.lang.NullPointerException: Cannot invoke "java.util.logging.Logger.log(java.util.logging.Level, String, java.lang.Throwable)" because "me.filoghost.chestcommands.fcommons.logging.Log.logger" is null
at ChestCommands.jar//me.filoghost.chestcommands.fcommons.logging.Log.warning(Log.java:32)
at ChestCommands.jar//me.filoghost.chestcommands.fcommons.logging.Log.warning(Log.java:28)
at ChestCommands.jar//me.filoghost.chestcommands.ChestCommands.onCheckedEnable(ChestCommands.java:70)
at ChestCommands.jar//me.filoghost.chestcommands.fcommons.BaseJavaPlugin.onEnable(BaseJavaPlugin.java:21)
at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264)
at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370)
at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500)
at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugin(CraftServer.java:535)
at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugins(CraftServer.java:449)
at org.bukkit.craftbukkit.v1_17_R1.CraftServer.reload(CraftServer.java:970)
at org.bukkit.Bukkit.reload(Bukkit.java:769)
at org.bukkit.command.defaults.ReloadCommand.execute(ReloadCommand.java:54)
at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159)
at org.bukkit.craftbukkit.v1_17_R1.CraftServer.dispatchCommand(CraftServer.java:838)
at net.minecraft.server.network.PlayerConnection.handleCommand(PlayerConnection.java:2257)
at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:2068)
at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:2049)
at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:46)
at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:6)
at net.minecraft.network.protocol.PlayerConnectionUtils.lambda$ensureRunningOnSameThread$1(PlayerConnectionUtils.java:56)
at net.minecraft.server.TickTask.run(TickTask.java:18)
at net.minecraft.util.thread.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:149)
at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.executeTask(IAsyncTaskHandlerReentrant.java:23)
at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:1418)
at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:192)
at net.minecraft.util.thread.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:122)
at net.minecraft.server.MinecraftServer.bf(MinecraftServer.java:1396)
at net.minecraft.server.MinecraftServer.executeNext(MinecraftServer.java:1389)
at net.minecraft.util.thread.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:132)
at net.minecraft.server.MinecraftServer.sleepForTick(MinecraftServer.java:1367)
at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1278)
at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319)
at java.base/java.lang.Thread.run(Thread.java:831)
The plugin has been disabled.
```

### Chest Commands version

4.0.3

**Comment by AnimeWatcherX** [comment]
Yes, I am also having this issue, 4.0.3. Idk what the issue is and its really annoying.

**Comment by GEADR** [comment]
+ help

**Comment by lehuymanhtan** [comment]
I got this error, but when I stop then start it again, the error disappears.

**Comment by filoghost** [comment]
Fixed in 4.0.4.
